screens/SettingsScreen.styled.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components/native';
+
+const StyledView = styled.View`
```

Comment:
I see this type of style is repeated several times in this PR. Do you think we can have common UI view styles, like: `StyledCenteredView`, `StyledCenteredViewWIthBackground`?